src/bandersnatch/main.py

```diff
@@ -102,7 +102,7 @@ async def async_main(args: argparse.Namespace, config: ConfigParser) -> int:
         if status_file.exists():
             tmp_status_file = Path(gettempdir()) / "status"
             try:
-                shutil.move(status_file, tmp_status_file)
+                shutil.move(str(status_file), tmp_status_file)
```

Comment:
Hmm - I thought this could handle a path-like-object but both 3.6 and 3.7 documentation don't say it can. So thanks!

Reply:
Relevant (it's interesting): https://github.com/python/typeshed/pull/3559